src/test/java/io/orten/nano/OrganizationTest.java

```diff
@@ -31,8 +31,8 @@ public void before() {
             }));
     }
 
-    @Test
-    public void saveorganaization()
+    /*@Test
+     public void saveorganaization()
```

Comment:
I don't like that you are removing testcases, but in this case, it is not testing anything so it is OK.
However there shouldn't be any commented out code in the master branch, so this should all be deleted.
You can always go back to a previous version in git if you need to see what the code looked like.